packages/components/src/components/post-popovercontainer/post-popovercontainer.scss

```diff
@@ -35,12 +35,16 @@
     max-width: 100%;
     color: var(--post-current-palette-fg);
     background-color: var(--post-current-palette-bg);
-    border: 2px solid transparent; // Keeping the default border for HCM
     border-radius: commons.$border-radius;
-    
+    overflow: hidden;
```

Comment:
This creates a bug in the post-popup and post-tooltip components, if you want to show them with an pointy arrow.